packages/terra-core-docs/src/terra-dev-site/doc/pills/example/BasicRemovablePill.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import PillList, { Pill } from 'terra-pills';
+
+const BasicRemovablePill = () => {
+  const [isRemoved, setIsRemoved] = useState(true);
```

Comment:
It might be nice to add a button to reset this example's state so that it can be manipulated multiple times.